src/less/overwrites/_alert.less

```diff
@@ -10,14 +10,18 @@
 // </ul>
 //
 // Markup:
-// <div class="alert alert-success alert-dismissible" role="alert">
-//  <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-//    Well done! You successfully read this important alert message. <a href="#">Link</a>
-// </div>
-// <div class="alert alert-danger alert-dismissible" role="alert">
-//  <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-//   Stop! This is all wrong. <a href="#">Link</a> 
-// </div>
+//  <div class="alert alert-success alert-dismissible" role="alert">
```

Comment:
Let's change these to use the KSS Modifiers and do a list of all of the possible classes while we're in here.